src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/src/styles/index.scss">
   </head>
   <body>
     <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="/src/images/imac.jpeg"
+        alt="photo imac"
+        >
```

Comment:
```suggestion
      <img
        class="card__image"
        src="/src/images/imac.jpeg"
        alt="photo imac"
      >
```

small issue with formatting, check in all places